### DISCUSSION TITLE
Missing VMX Packager Lambda Logs/Cases

### DISCUSSION THREAD

**Discussion by enilvia** [amazon-connect/voicemail-express-amazon-connect#16] [General]
I have updated my contact center to the latest version. Something seems to be broken. Some cases are being created others are not. All Voice recordings are in the S3 bucket in some instance there are no Lambda logs or case in Salesforce. However the call log exist

**Comment by dougjaso** [comment]
Same as issue opened. This version does not create cases in Salesforce. You need to [use this solution](https://github.com/amazon-connect/amazon-connect-salesforce-scv/tree/master/Solutions/VMX2-VoicemailExpress).
